apps/backend/src/main.ts

```diff
@@ -1,7 +1,7 @@
 import { ValidationPipe } from '@nestjs/common';
 import { NestFactory } from '@nestjs/core';
 import { AppModule } from './app.module';
-import * as cookieParser from 'cookie-parser';
```

Comment:
I had to change the cookie parser import after changing the module because there was an error with that